content/01.main_content.md

```diff
@@ -1,43 +1,45 @@
+  After this study was publicized, ISCB's following honorees had the highest mean predicted probability of having an East Asian name of any previous year, and the nominating committee inducted their first China-based ISCB Fellow [@doi:10.1016/j.cels.2021.07.007].
 
 Audits can also provide evidence for specific actions.
   A recent study of disparities in scientific journalism found that predicted East Asian names were under-quoted and under-mentioned [@doi:10.1101/2021.06.21.449261].
+  <!--I think we want to be pretty careful here unless we're sure these particular results are statistically robust -->
   Interestingly, this disparity differed across the predicted name-origins of journalists.
   Furthermore, the differences between journalists was almost completely removed when considering people with a US-affiliated paper cited in the article.
   This suggests a regional bias, arguing for more regionally focused journalists.
+  <!--Maybe remove the above?-->
 
 Ideally, audits would only use self-reported gender, ethnicity, and other identifications [@doi:10.1371/journal.pcbi.1003903].
   While this is possible prospectively, surveying is impractical for large groups and often impossible retrospectively [@http://mediashift.org/2014/11/how-to-ethically-and-responsibly-identify-gender-in-large-datasets/].
-  In contrast, computationally-derived predictions allow for large-scale audits with as broad a scope as needed.
+  Computationally-derived predictions allow for large-scale audits with as broad a scope as needed.
   Numerous tools exist to algorithmically infer gender, nationality, and ethnicity using the feature most likely to be present in datasets: an individual's name [@doi:10.1093/pan/mpw001; @http://genderize.io/].
-  
+
 Prediction models are not a panacea; several factors limit both their accuracy and utility.
   For instance, gender associations of a given name can vary by culture, potentially biasing gender predictions where additional information is not available [@https://aclanthology.org/U14-1021].
-  Also, most gender prediction models are trained on binary gender labels, which occludes assessing the representation of transgender, non-binary, and intersex individuals [@doi:10.1101/2020.10.12.336230].
- 
-Proxy predictions of ethnicity via name origin are more difficult still; choosing categories to probabilistically predict on is non-trivial and difficult to discretize. 
+  Also, most gender prediction models are trained on binary gender labels, which occludes assessing and the representation of transgender, non-binary, and intersex individuals [@doi:10.1101/2020.10.12.336230].
+  <!--I think the concerns with the above aren't just that it makes it impossible to assess but also that it may conceptually erase the experiences of such folks -->
```

Comment:
Totally agree, and I'm wondering if there's a way to acknowledge that without costing many words. Maybe "Also, most gender prediction models are trained on binary gender labels, which systematically overlooks representation of transgender, non-binary, and intersex individuals"?